nilearn/plotting/surf_plotting.py

```diff
@@ -1123,13 +1126,18 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
     # Call _get_colorbar_and_data_ranges to derive symmetric vmin, vmax
     # And colorbar limits depending on symmetric_cbar settings
     cbar_vmin, cbar_vmax, vmin, vmax = _get_colorbar_and_data_ranges(
-        loaded_stat_map, vmax, symmetric_cbar, kwargs)
+        loaded_stat_map,
+        vmin=vmin,
+        vmax=vmax,
+        symmetric_cbar=symmetric_cbar,
+        symmetric_data_range=False,
+    )
 
     display = plot_surf(
         surf_mesh, surf_map=loaded_stat_map,
         bg_map=bg_map, hemi=hemi,
         view=view, engine=engine, avg_method='mean', threshold=threshold,
-        cmap=cmap, symmetric_cmap=True, colorbar=colorbar,
+        cmap=cmap, symmetric_cmap=symmetric_cbar, colorbar=colorbar,
```

Comment:
```suggestion
        cmap=cmap, symmetric_cmap=symmetric_cmap, colorbar=colorbar,
``` (not sure I understand)

Reply:
I also find this confusing. [`symmetric_cmap`](https://github.com/search?q=repo%3Anilearn%2Fnilearn+symmetric_cmap&type=code) and [`symmetric_cbar`](https://github.com/search?q=repo%3Anilearn%2Fnilearn+symmetric_cbar&type=code) are used in different functions, and it is unclear to me if they are supposed to be the same thing. 

After looking into this a little more, I think I should change this back to what it was before (`symmetric_cmap=True`), since the [docs for `plot_surf`](https://github.com/mathdugre/nilearn/blob/44695d8c9c784d71bc2635650236859155089c0a/nilearn/plotting/surf_plotting.py#L679-L688) say that `symmetric_cmap` is only implemented for the `plotly` engine, and the `plot_img_on_surf` function I changed specifically does not handle plotting with Plotly.